src/bin.ts

```diff
@@ -199,6 +199,8 @@ export function main (argv: string[]) {
       : undefined
   })
 
+  const requires = argsRequire.length !== 0 ? argsRequire : service.options.require || []
```

Comment:
Should we merge these lists together? Seems odd to have the CLI entirely overwrite the options when specified.

Reply:
Yeah, that makes sense.  I can't think of a good use-case where you'd want the `--require` flag to suppress whatever was specified in the tsconfig.